inbm/dispatcher-agent/dispatcher/source/constants.py

```diff
@@ -25,22 +26,22 @@ class SourceParameters:
 
```

Comment:
**Consistency**
**Suggestion:** To maintain consistency and avoid potential bugs, consider initializing sources with a consistent default factory method as used in other classes.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
@dataclass(kw_only=True, frozen=True)
class ApplicationSourceList:
    sources: List[str] = field(default_factory=list)
```